### PR TITLE
Enhancement

### PR DESCRIPTION
**Pull request by jinningwang**
Several enhancements for following issues:

Equivalent star bus of three-winding transformer: https://github.com/CURENT/andes/issues/571

Clarify PFlow solution: https://github.com/CURENT/andes/issues/570

Add a method `get_all_idxes()` to group: https://github.com/CURENT/andes/issues/568,

Allow all matches in `find_idx()`: https://github.com/CURENT/andes/issues/566#issuecomment-2376950664

### CODE REVIEW

**Comment by jinningwang** [comment]
@cuihantao I think it's ready to merge

File: andes/models/group.py [hunk 1]
```diff
@@ -438,6 +438,32 @@ def get_next_idx(self, idx=None, model_name=None):
...
+        Examples
+        --------
+        >>> ss = andes.load(andes.get_case('ieee14/ieee14_pvd1.xlsx'))
+        >>> ss.DG.get_idx()
+        [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+
+        >>> ss.StaticGen.get_idx()
+        [2, 3, 4, 5, 6, 1]
+        """
+        all_idx = [mdl.idx.v for mdl in self.models.values()]
```
Comment (cuihantao) [inline andes/models/group.py]: How is this different from `list(self._idx2models.keys())`?

File: andes/routines/pflow.py [hunk 2]
```diff
@@ -16,6 +16,9 @@
 class PFlow(BaseRoutine):
     """
     Power flow calculation routine.
+
+    In power flow analysis, there are reactive power limits for PV generators (see :ref:`PV`), but it does
```
Comment (cuihantao) [inline andes/routines/pflow.py]: This statement lacks clarity. 

"there are reactive power limits" is ambiguous regarding whether it is turned on by default or not.

You can say "Power flow analysis currently supports limiting reactive power (needs to to be turned on via `config.pv2pq`) but does not enforce voltage limits."

Comment (cuihantao) [reply andes/routines/pflow.py]: And maybe no need to say the last sentence. That's a  may be.

File: andes/models/group.py (hunk 1 above)
Comment (cuihantao) [reply andes/models/group.py]: Maybe rename this to `get_all_idxes`? The name `get_idx` might imply it's getting the `idx` for some inputs, but this function takes no input.

**Comment by jinningwang** [comment]
Have addressed your concerns accordingly.

**Comment by jinningwang** [comment]
@cuihantao any more suggestions? If looks good we can merge it.

**Comment by cuihantao** [comment]
Thanks! Merging in a second.

**Pull request merged**